packages/@react-spectrum/combobox/test/ComboBox.test.js

```diff
@@ -1479,6 +1623,38 @@ describe('ComboBox', function () {
 
       expect(onSelectionChange).toHaveBeenCalledTimes(1);
       expect(onSelectionChange).toHaveBeenCalledWith(null);
+
+      listbox = getByRole('listbox');
+      expect(listbox).toBeVisible();
+      items = within(listbox).getAllByRole('option');
+      expect(items[1]).toHaveTextContent('Two');
+      expect(items[1]).toHaveAttribute('aria-selected', 'true');
+    });
+
+    it('does not fire onSelectionChange if both inputValue and selectedKey are controlled', function () {
+      let {getByRole, rerender} = render(<ExampleComboBox inputValue="Two" selectedKey="2" />);
+
+      let combobox = getByRole('combobox');
+      act(() => {
+        combobox.focus();
+        fireEvent.keyDown(combobox, {key: 'ArrowDown', code: 40, charCode: 40});
+        fireEvent.keyUp(combobox, {key: 'ArrowDown', code: 40, charCode: 40});
```

Comment:
Do we need these here? Don't think we do anything with the menu here, though it would be nice if we checked that selecting an item doesn't fire onInputChange (I believe that is how this works)